### DISCUSSION TITLE
packagekit not installed

### DISCUSSION THREAD

**Discussion by Razriel3** [cockpit-project/cockpit#18903] [Q&A]
I am working with YOCTO to build my distro using cockpit as linux managment tool.
All needed capabilities are working as expected. 
only problem left is that I am getting a error sign in the system overview health frame.
saying that "Loading available updates failed"
when clicking on the link "PackageKit is not installed" 
I dont wont to have this capability. I am not providing my user package managment capabilities. 
how can I remove this feature ?

Thanks,

Roy

**Comment by marusak** [comment]
I think you have `cockpit-packagekit` installed? If you remove this plugin you should not see this alert.
